File: dotnet/PhysiciansReach/PR.Models/UserAccountModel.cs
using System;

namespace PR.Models
{
    public class UserAccountModel
    {
        public int UserAccountId { get; set; }

        public string UserName { get; set; }

        public string Password { get; set; }

        public string ConfirmationPassword { get; set; }

        public DateTime CreatedOn { get; set; }

        public DateTime ModifiedOn { get; set; }

        public ErrorModel error { get; set; }
    }
}

File: angular/src/app/models/user-account.model.ts
import { AccountType } from './enums/account-type.enum';
import { ErrorModel } from './error.model';

export class UserAccount {
    public userAccountId: string;
    public type: AccountType;
    public userName: string;
    public password: string;
    public emailAddress: string;
    public active: boolean;
    public confirmationPassword: string;
    public errors: ErrorModel[];
}
